### DISCUSSION TITLE
delivery point should be a standalone point based on its tags

### DISCUSSION THREAD

**Discussion by noredinebahri** [openstreetmap/iD#9201] [General]
hi,
I expect to see a **node point,** but I see a **node vertex** and the warning is: 

![image](https://user-images.githubusercontent.com/3017515/178962426-2f12cf4e-57ea-42bc-af17-77924230bd88.png)

**Comment by tyrasd** [comment]
Can you please share which tags this node has or had? Without it it is very had to look into the possible causes of this issue.

**Reply by tyrasd** [reply]
This is likely caused by a preset which is limited to the `geometry` type `point` (which is defined as a node which is not part of a way). When the respective tag is preset on a `vertex` (i.e. on a node which is part of a way), the warning is triggered because iD thinks that the respective preset is not supposed to be used on a `vertex`.

The easiest fix would be to allow the preset to be applied on all nodes:

```json
    "geometry": [
        "point",
        "vertex"
    ],
```
